tests/tests/runner.rs

```diff
@@ -151,3 +168,57 @@ async fn file_data_sources() {
     let stop_block = test_ptr(2);
     ctx.start_and_sync_to(stop_block).await;
 }
+
+#[tokio::test]
+async fn template_static_filters_false_positives() {
+    let stores = stores("./integration-tests/config.simple.toml").await;
+
+    let subgraph_name = SubgraphName::new("dynamic-data-source").unwrap();
+    // let subgraph_name = SubgraphName::new("non-fatal-errors").unwrap();
+    let hash = {
+        let test_dir = format!("./integration-tests/{}", subgraph_name);
+        fixture::build_subgraph(&test_dir).await
+    };
+
+    let blocks = {
+        let block_0 = genesis();
+        let block_1 = empty_block(block_0.ptr(), test_ptr(1));
+        let block_2 = empty_block(block_1.ptr(), test_ptr(2));
+        vec![block_0, block_1, block_2]
+    };
```

Comment:
Have you checked that this test fails on master? I'm surprised that it would reproduce with just empty blocks.

Reply:
```
Sep 21 10:10:07.912 DEBG 1 candidate trigger in this block, block_hash: 0x0000000000000000000000000000000000000000000000000000000000000001, block_number: 1, sgd: 9, subgraph_id: Qmepz1JnzZNFHLhYRUBLgi53dnhuukEuKNjCKzsg2g1YrF, component: SubgraphInstanceManager
```

This is printed during the test and is being filtered out by match_and_decode, it's the trigger we need.

Reply:
Nice.